.gitignore

```diff
@@ -35,7 +35,7 @@ pip-delete-this-directory.txt
 # Unit test / coverage reports
 htmlcov/
 .tox/
-.coverage
+.coverage*
```

Comment:
This change would cuase `.coveragerc` to be ignored. It should be reverted I think.